sprangu-front/src/app/app.component.css

```diff
@@ -1,10 +0,0 @@
-@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css");
-
-.app {
-  padding-left: 20%;
-  padding-right: 20%;
-}
-
-.img-wrapper {
-  height: 250px;
-}
```

Comment:
Kodėl šitą ištrynei? jis yra keliuose html'uose naudojamas

Reply:
Buvo tik prie tools for rental kompnente tai tik ten ir pridėjau